src/main/ml-config/security/roles/geo-data-services-model-cache-role.json

```diff
@@ -12,6 +12,11 @@
       "privilege-name":"xdmp:set-server-field",
       "action": "http://marklogic.com/xdmp/privileges/xdmp-set-server-field",
       "kind": "execute"
+    },
+    {
+      "privilege-name": "xdmp:get-session-field",
```

Comment:
This is a bummer. This role is being used for amps, but you'd never guess it by the role name. I'd like for this privilege to eventually be assigned via the "internal" role, but that would require a separate "amped-functions.xqy" module since the code that calls this is in XQuery, not SJS. Probably a cleanup effort in a few days.